Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -697,3 +697,22 @@ extension AVFoundationPlayback {
         return 5
     }
 }
+
+// MARK: - Metrics
+extension AVFoundationPlayback {
+    public func getBitrate() -> Double? {
+        guard let logEvent = player?.currentItem?.accessLog()?.events.last else { return nil }
```

Comment:
how about having a function that returns the last `AVPlayerItemAccessLogEvent`? So we can reuse it in both `getBitrate` and `getAvgBitrate` functions. Something like this:
```swift
private func lastLogEvent() -> AVPlayerItemAccessLogEvent? {
    return player?.currentItem?.accessLog()?.events.last
}

public func getBitrate() -> Double? {
    guard let logEvent = lastLogEvent() else { return nil }
    if logEvent.segmentsDownloadedDuration > 0 {
        return logEvent.indicatedBitrate
    }
    return logEvent.observedBitrate
}

public func getAvgBitrate() -> Double? {
    guard let logEvent = lastLogEvent else { return nil }
    if #available(iOS 10.0, *) {
        return logEvent.averageVideoBitrate
    }
    return nil
}
```